### ISSUE TITLE
Loudness information over OSC

### ISSUE THREAD

**Issue by baltedewit** [CasparCG/server#1301]
### Description
To make loudness information available over OSC for implementation by clients in their UI for both individual producers as well as channels.

### Solution suggestion
Can be done by using https://github.com/jiixyj/libebur128 I think

---

### Additional information
This would be an example project of where it would be nice to have this information for the producer: https://github.com/olzzon/sisyfos-audio-controller

**Comment by herzogmedia** [comment]
sounds interesting, and could be valuable for a lot of things. 
Just to clarify, what info would you think of? the integrated loundess values for already scanned clips in the media folder? "realtime"-info like short-term or momentary? the preliminary integrated value from clip start to current time?

**Comment by baltedewit** [comment]
Integrated loudness for clips is not something I'd add in the server, but rather let mediascanner handle that. (It's definitely on my list of things to add there)

I'm more interested in realtime short and momentary values. This is currently not achievable in the box for live inputs and outputs without adding additional things into the chain. I have been working on a project where all the mixing, including audio, is done inside the box and it would be nice to have metering in the box as well.

Integrated values for nice to have, _but_ it would require an additional API for resetting them.

**Comment by gizahNL** [comment]
Iirc ebur128 adds some delay

**Comment by baltedewit** [comment]
> Iirc ebur128 adds some delay

In that case it should be fed in parallel to the regular mixer pipeline.
